novawallet/Modules/Staking/Model/StakingManageOption.swift

```diff
@@ -91,11 +91,15 @@ enum StakingManageOption {
         }
     }
 
-    static func proxyAction(from proxyDefinition: ProxyDefinition?) -> StakingManageOption {
-        guard let proxiesCount = proxyDefinition?.definition.count, proxiesCount > 0 else {
+    static func proxyAction(from proxyDefinition: UncertainStorage<ProxyDefinition?>) -> StakingManageOption? {
+        guard proxyDefinition.isDefined else {
```

Comment:
I would check hasProxy in the chain to determine whether to display addProxy action. This is because we might switch off proxy support but the definition still remain in cache.